### PR TITLE
update CI to GHA

### PR DESCRIPTION
**Pull request by mileslucas**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          - windows-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - uses: actions/cache@v1
```
Comment (giordano) [inline .github/workflows/ci.yml:32]:
```suggestion
      - uses: actions/cache@v2
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - uses: julia-actions/julia-buildpkg@v1
+      - uses: julia-actions/julia-runtest@v1
+      - uses: julia-actions/julia-processcoverage@v1
+      - uses: codecov/codecov-action@v1
+        with:
+          file: lcov.info
```
Comment (giordano) [inline .github/workflows/ci.yml:47]:
```suggestion
          file: lcov.info

```

File: .github/workflows/preview-cleanup.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+name: Doc Preview Cleanup
```
Comment (giordano) [inline .github/workflows/preview-cleanup.yml:1]: Cool, I didn't know this!

Comment (giordano) [reply .github/workflows/preview-cleanup.yml:1]: Do you need an SSH key (like `DOCUMENTER_KEY`) for this to work?

Comment (mileslucas) [reply .github/workflows/preview-cleanup.yml:1]: doesn't seem like it just by skimming. I yoinked it from MCMCChains originally.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
-SkyCoords.jl
-============
+# SkyCoords.jl
 
-Basic astronomical coordinate systems in Julia
+[![Build Status](https://github.com/turinglang/SkyCoords.jl/workflows/CI/badge.svg?branch=master)](https://github.com/JuliaAstro/SkyCoords.jl/actions)
```
Comment (giordano) [inline README.md:3]: Direct link to the CI workflow:
```suggestion
[![Build Status](https://github.com/turinglang/SkyCoords.jl/workflows/CI/badge.svg?branch=master)](https://github.com/JuliaAstro/SkyCoords.jl/actions/workflows/ci.yml)
```

File: docs/src/index.md [hunk 5]
```diff
@@ -1,8 +1,9 @@
 # SkyCoords.jl
 
-[![Build Status](https://img.shields.io/travis/JuliaAstro/SkyCoords.jl.svg)](https://travis-ci.org/JuliaAstro/SkyCoords.jl)
-[![Build status](https://img.shields.io/appveyor/ci/kbarbary/skycoords-jl.svg?label=windows)](https://ci.appveyor.com/project/kbarbary/skycoords-jl/branch/master)
-[![codecov](https://codecov.io/gh/JuliaAstro/SkyCoords.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaAstro/SkyCoords.jl)
+[![GitHub](https://img.shields.io/badge/Code-GitHub-black.svg)](https://github.com/JuliaAstro/SkyCoords.jl)
+[![Build Status](https://github.com/JuliaAstro/SkyCoords.jl/workflows/CI/badge.svg?branch=master)](https://github.com/JuliaAstro/SkyCoords.jl/actions)
```
Comment (giordano) [inline docs/src/index.md:4]:
```suggestion
[![Build Status](https://github.com/JuliaAstro/SkyCoords.jl/workflows/CI/badge.svg?branch=master)](https://github.com/JuliaAstro/SkyCoords.jl/actions/workflows/ci.yml)
```

**Review by giordano — APPROVED** [review]

**Pull request merged**